src/main/java/wordle/model/Wordle.java

```diff
@@ -0,0 +1,89 @@
+package wordle.model;
+
+import java.time.LocalDate;
+import wordle.view.Console;
+
+public class Wordle {
+
+    private static final int TRY_COUNT_LIMIT = 6;
+
+    private final Console console;
+    private final WordService wordService;
+    private final WordleValidator wordleValidator;
+    private final TileService tileService;
+
+    public Wordle(Console console, WordService wordService, WordleValidator wordleValidator, TileService tileService) {
+        this.console = console;
+        this.wordService = wordService;
+        this.wordleValidator = wordleValidator;
+        this.tileService = tileService;
+    }
+
+    public void start() {
+        console.printInitGameMessage();
+
+        Words words = wordService.getWords();
+        Letters answerLetters = createAnswerLetters(words);
```

Comment:
이 부분은 위에서 언급했던 Words를 매개변수로 선언했다는 가정하에 리뷰해주시면 좋을 거 같습니다☺️

```suggestion
        Letters answerLetters = wordService.createAnswerLetters();
```
단어 목록을 로드하고 답변을 만들어 내는 책임을 wordService에 위임하는 방식은 어떨까요?
만약 단어를 로드하는 방식을 수정하는 요구사항이 생기면 Wordle의 createAnswerLetter() 부분이 변경될 수도 있는데
단어를 로드하는 책임이 온전히 wordService에게 있다면 SRP를 지키는 코드가 되지 않을까 싶습니다!
제가 선호하는 방식이라 자유롭게 의견을 들어보고 싶네요!☺️

Reply:
마감 이슈로 일단 메서드 분리만 해보자해서 내부 메서드로 추출하고 개선은 못했는데 위에서 의견주신 것과 동일하게 위임하면 좋을 것 같아요 ㅎㅎ 좋은 의견 감사합니다 👍🏻